Makefile

```diff
@@ -298,16 +298,37 @@ test-go:
 # Note: to disable npm tests set `NO_NPM` environment variable to `TRUE`.
 .PHONY: integration-test
 integration-test:
-	scripts/dev_scripts/integration_tests.sh $(REPO_PATH) "${HOME}"
+	if [ "${NO_NPM}" == "TRUE" ]; then \
+    	echo "Note: NO_NPM environment variable is set to TRUE, so npm tests will be skipped."; \
+		python ./tests/integration/run.py \
+			run \
+			--exclude-tag docker-only \
+			--exclude-tag skip \
+			--exclude-tag npm-registry \
+			./tests/integration/cases/...; \
+	else \
+		python ./tests/integration/run.py \
+			run \
+			--exclude-tag docker-only \
+			--exclude-tag skip \
+			./tests/integration/cases/...; \
+	fi
 
 .PHONY: integration-test-docker
 integration-test-docker:
-	scripts/dev_scripts/integration_tests_docker.sh $(REPO_PATH) scripts/release_scripts/run_macaron.sh
+	python ./tests/integration/run.py \
+		run \
+    	--macaron scripts/release_scripts/run_macaron.sh \
+		--include-tag docker-only \
+    	--include-tag shared-docker-python \
+    	./tests/integration/cases/...
```

Comment:
There seems to be tab/spaces issues here. I will address them.

Reply:
Addressed in https://github.com/oracle/macaron/pull/782/commits/0defbf663113b242a043603e0c8fd6e56b5797be